repo/packages/K/kafka/5/package.json

```diff
@@ -7,8 +7,8 @@
   "description": "Apache Kafka running on top of Apache Mesos",
   "framework": true,
   "tags": ["message", "broker", "messaging"],
-  "preInstallNotes": "This will install Apache Kafka DCOS Service.",
-  "postInstallNotes": "The Apache Kafka DCOS Service is installed.",
+  "preInstallNotes": "You are about to install DCOS Kafka Service.",
+  "postInstallNotes": "DCOS Kafka Service is installing.",
```

Comment:
Just a suggestion:
Other alternative notes:
- `DCOS Kafka Service is getting installed`
- `Installation of DCOS Kafka Service is in progress`